### DISCUSSION TITLE
Pass Common Restriction (commonRestriction/commonValue) to the Subquery

### DISCUSSION THREAD

**Discussion by Monshou1251** [ariacom/Seal-Report#114] [General]
The question is pretty simple: if it is possible to pass a Restriction from the Parent Report to the Subreport, is it also possible to pass commonRestriction or commonValues?

**Comment by ariacom** [comment]
It seems not directly supported...
Please check https://github.com/ariacom/Seal-Report/discussions/103

**Reply by Monshou1251** [reply]
Thank you for prompt reply!
